tests/components/husqvarna_automower/test_button.py

```diff
@@ -92,6 +93,55 @@ async def test_button_states_and_commands(
         )
 
 
+@pytest.mark.freeze_time(datetime.datetime(2024, 2, 29, 11, tzinfo=datetime.UTC))
+async def test_sync_clock(
+    hass: HomeAssistant,
+    mock_automower_client: AsyncMock,
+    mock_config_entry: MockConfigEntry,
+    freezer: FrozenDateTimeFactory,
+) -> None:
+    """Test button commands."""
+    entity_id = "button.test_mower_1_sync_clock"
+    await setup_integration(hass, mock_config_entry)
+    state = hass.states.get(entity_id)
+    assert state.name == "Test Mower 1 Sync clock"
+
+    values = mower_list_to_dictionary_dataclass(
+        load_json_value_fixture("mower.json", DOMAIN)
+    )
+    mock_automower_client.get_status.return_value = values
+
+    await hass.services.async_call(
+        domain="button",
+        service=SERVICE_PRESS,
+        target={ATTR_ENTITY_ID: entity_id},
```

Comment:
```suggestion
        BUTTON_DOMAIN,
        SERVICE_PRESS,
        {ATTR_ENTITY_ID: entity_id},
```
`from homeassistant.components.button import DOMAIN as BUTTON_DOMAIN`

Reply:
done in https://github.com/home-assistant/core/commit/ef7a14c3a8ffe3964f796a47d705b5bc5ba8b8d7 and https://github.com/home-assistant/core/pull/125689/commits/605ef2d048adcee7620f023291767f47ea36b2f6